java/src/jmri/jmrix/openlcb/swing/stleditor/StlEditorPane.java

```diff
@@ -899,6 +899,7 @@ private void updateDescription() {
             if (count < termCount) {
                 count += addToDescription(ident.getSoftwareVersion(), sb);
```

Comment:
Is that really a false warning?  `count` is being stored in line 900, and I don't see a reference to it after that.